release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2740,6 +2740,53 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.31 --pullspecs
+----
+
+[id="ocp-4-15-31-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the `spec.noProxy` field from the cluster-wide proxy was not considered when the {cmo-first} configured proxy capabilities for a Prometheus remote write endpoint. With this release, the {cmo-short} no longer configures proxy capabilities for any remote write endpoints whose URL would bypass the proxy according to the `noProxy` field. (link:https://issues.redhat.com/browse/OCPBUGS-39172[*OCPBUGS-39172*])
+
+* Previously, utlization cards displayed limit in a way that incorrectly implied a relationship between capacity and limits. With this release, the position of limit is changed to remove this implication. (link:https://issues.redhat.com/browse/OCPBUGS-39085[*OCPBUGS-39085*])
+
+* Previously, the `openvswitch` service used older cluster configurations after a cluster upgrade and this caused the `openvswitch` service to stop. With this release, the `openvswitch` service is now restarted after a cluster upgrade so that the service uses the newer cluster configurations. (link:https://issues.redhat.com/browse/OCPBUGS-34842[*OCPBUGS-34842*])
+
+* Previously, after you submitted the same value into the {vmw-first} configuration dialog, cluster nodes unintentionally rebooted. With this release, nodes reboot after you enter new values into the dialog and not the same values.(link:https://issues.redhat.com/browse/OCPBUGS-33938[*OCPBUGS-33938*])
+
+* Previously, if a virtual machine (VM) was deleted and the network interface controller (NIC) still existed for that VM, the {azure-first} VM verification check crashed. With this release, the verification check can now handle this situation by gracefully processing the issue without crashing. (link:https://issues.redhat.com/browse/OCPBUGS-31467[*OCPBUGS-31467*])
+
+* Previously, Red{nbsp}Hat HyperShift periodic conformance jobs failed because of changes to {op-system-first}. With this release, an update to the trusted certificate authority (CA) certificates fixes the issue so that periodic conformance jobs can succeed.(link:https://issues.redhat.com/browse/OCPBUGS-38943[*OCPBUGS-38943*])
+
+* Previously, the grace period for a node to become ready was not aligned with upstream, and in some situations, the previous set period made the node cycle between `Ready` and `Not ready`. (link:https://issues.redhat.com/browse/OCPBUGS-39077[*OCPBUGS-39077*])
+
+[id="ocp-4-15-31-known-issues_{context}"]
+==== Known issues
+
+* On {product-rosa}, node pools might stop scaling workloads or updating configurations for a cluster that uses the {product-version}.23 or later version of the {hcp-capital} service. Depending on the version of components that interact with your cluster, you can resolve this issue be completing the steps in one of the following Knowledgebase articles:
++
```

Comment:
For the previous paragraph, the SSG says use the two-word "knowledge base" unless referring specifically to the "Red Hat Knowledgebase".

https://redhat-documentation.github.io/supplementary-style-guide/#knowledge-base

Reply:
:heavy_check_mark: